ontology/yaml/resources/HVAC/entity_types/VAV.yaml

```diff
@@ -388,6 +388,14 @@ VAV_DSP_ED:
   - DSP
   - ED
 
+VAV_CO2C_DDPM:
+  description: "Discharge control VAV with CO2 regulation and damper percentage monitoring"
+  is_canonical: true
+  implements:
+  - VAV
+  - CO2C
+  - DDPM
```

Comment:
In this context, the appropriate type is PDSCV. The new type is unnecessary.

Reply:
Ok. I created the new type because the air is directly blowed into the room. I will remove the new type and change the VAV type accordingly.